language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/DamlLedgerClient.java

```diff
@@ -24,13 +23,12 @@
  *     <li>Call the method {@link DamlLedgerClient#connect()} to initialize the clients for that particular ledger</li>
  *     <li>Retrieve one of the clients by using a getter, e.g. {@link DamlLedgerClient#getActiveContractSetClient()}</li>
  * </ol>
- *
+ * <p>
```

Comment:
I'm not sure of what is considered a good approach in JavaDoc, but for my limited knowledge of HTML I'd prefer either making full paragraphs or use `<br /><br />`. Feel free to make it part of a separate PR if you think it's too much work to sneak into this one.

Reply:
The IntelliJ autoformatter inserted these, so I'm inclined to just roll with it.

Reply:
Seems to in-line with the docs: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format.

Reply:
I will contain my surprise at Oracle suggesting that non-well-formed XML is the way to go and quietly mark this as solved.